Code/GraphMol/RGroupDecomposition/Wrap/rdRGroupComposition.cpp

```diff
@@ -195,6 +195,26 @@ void relabelMappedDummiesHelper(ROMol &mol, unsigned int inputLabels,
                        static_cast<RGroupLabelling>(outputLabels));
 }
 
+ROMol *rgroupRowZipHelper(python::dict row, const MolzipParams &p) {
```

Comment:
Because the python wrapper doesn't need to know about the RGroupRow, this function could easily exist in the ChemTransforms wrapper and might be the right thing to do because it shares the same name "molzip" as the code there.

This way, you wouldn't need to load molzip from the rdRGroupComposition.  Thoughts?

Reply:
I think it would make sense to do it this way.

Reply:
are you planning on making this change in this PR or should it wait for another one?

Reply:
I’ll make the change